### PR TITLE
Pub/sub code review

### PR DESCRIPTION
**Pull request by shalin1738**
Made the required changes from yestarday

### CODE REVIEW

**Comment by mdspielman** [comment]
Shalin, can you fix the merge conflicts and resubmit.

File: src/pubsub/index_debug.html [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+                        <h3 class="demoHeaders">Messages  <label id="stats_rxMsgCount">(0)</label> 
+                        </h3>
+                    </td>
+                </tr>
+            </table>
+        </td>
+    </tr>
+    <tr>
+        <td style="padding-top: 10px">
+            <h4>Copyright 2010-2016 Solace Systems, Inc.</h4>
```
Comment (mdspielman) [inline src/pubsub/index_debug.html]: Can you update this with the open source licences.

File: src/pubsub/index.html [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+                        <h3 class="demoHeaders">Messages  <label id="stats_rxMsgCount">(0)</label> 
+                        </h3>
+                    </td>
+                </tr>
+            </table>
+        </td>
+    </tr>
+    <tr>
+        <td style="padding-top: 10px">
+            <h4>Copyright 2010-2016 Solace Systems, Inc.</h4>
```
Comment (mdspielman) [inline src/pubsub/index.html]: This shouldn't be present once you update to Apache license.

File: src/pubsub/index.html [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+            <h6>
+
+/*
```
Comment (mdspielman) [inline src/pubsub/index.html]: This should not be within comments. See original format for ideas. Need to strip out the /* */ style comments.

File: src/pubsub/sol-direct-pubsub-ui.js [hunk 4]
```diff
@@ -0,0 +1,285 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
+// Solace Systems Messaging API for JavaScript
+// Copyright 2010-2016 Solace Systems, Inc.
```
Comment (mdspielman) [inline src/pubsub/sol-direct-pubsub-ui.js]: Update to Apache license

File: src/pubsub/sol-direct-pubsub.js [hunk 5]
```diff
@@ -0,0 +1,408 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
+// Solace Systems Messaging API for JavaScript
+// Copyright 2010-2016 Solace Systems, Inc.
```
Comment (mdspielman) [inline src/pubsub/sol-direct-pubsub.js]: Update to apache license

File: src/pubsub/sol-direct-pubsub-ui.js [hunk 6]
```diff
@@ -0,0 +1,284 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
+// Solace Systems Messaging API for JavaScript
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (mdspielman) [inline src/pubsub/sol-direct-pubsub-ui.js]: We need the open source license here.

File: src/pubsub/sol-direct-pubsub.js [hunk 7]
```diff
@@ -0,0 +1,407 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
+// Solace Systems Messaging API for JavaScript
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy of
```
Comment (mdspielman) [inline src/pubsub/sol-direct-pubsub.js]: Need proper license here

**Comment by shalin1738** [comment]
Changes made

**Pull request merged**